corehq/apps/app_manager/models.py

```diff
@@ -1154,6 +1154,7 @@ def add_stuff_to_xform(self, xform, build_profile_id=None):
         xform.set_version(self.get_version())
         xform.add_missing_instances(app)
 
+    @memoized
     def render_xform(self, build_profile_id=None):
         xform = XForm(self.source)
```

Comment:
Yeah, I made a change to avoid multiple wraps. https://github.com/dimagi/commcare-hq/pull/30146/commits/361f6874484cd8c55e3a22d85d2abc5d4713707f

The difference wasn't that big, around 2 seconds of improvement on the eCHIS app.

Reply:
I ended up reverting this as there seems to be way more effort required and the order of timing improvement is not that high.

Reply:
OK, thanks for looking into it!